gradio/components/image.py

```diff
@@ -102,6 +102,8 @@ def __init__(
         show_share_button: bool | None = None,
         placeholder: str | None = None,
         show_fullscreen_button: bool = True,
+        webcam_height: int = 1440,
```

Comment:
Thanks @yinsumirage for the nice contribution! 

I prefer we instead just let developers specify the [media constraints](https://developer.mozilla.org/en-US/docs/Web/API/Media_Capture_and_Streams_API/Constraints#applying_constraints) as a python dictionary and we pass that into the get_stream function. It will be more flexible that way and we can add docstring examples for controlling the height and width specifically. I think we can call this parameter `webcam_constraints`.

BTW, you also need to specify the new props in `image/Index.svelte` and `image/ImageUploader.svelte` and pass them into the Webcam component.

BTW can you do this for video as well now that you're at it? 😊